wercker.yml

```diff
@@ -12,6 +12,12 @@ build:
         code: |
           mkdir container/  
           cp ./target/*.jar container/
+  # Notify about the step status to slack
+  after-steps:
+    - slack-notifier:
+        url: $SLACK_URL
+        channel: notifications
+        username: Wercker Event Notifier
```

Comment:
Did you change this file?

Reply:
it`s devops changes, pls see commits in pull request